snapshot/git/gitfiler/checkouter.go

```diff
@@ -49,12 +47,25 @@ func (c *Checkouter) Checkout(id string) (co snapshot.Checkout, err error) {
 		{"checkout", id},
 	}
 
+	if err := c.runGitCmds(cmds, repo); err != nil {
+		// try fetching for new commits before returning error
+		// takes a long time (~5 min)
```

Comment:
Eep! 5 minutes is long... Do you have a breakdown? Does it take that long if you ssh in?